pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -162,6 +168,17 @@ var _ = Describe("Node-labeller ", func() {
 		))
 	})
 
+	It("should add cpu tsc labels if cpu counter exists and its name is tsc", func() {
+		res := nlController.execute()
+		Expect(res).To(BeTrue())
+
+		node := retrieveNode(kubeClient)
+		Expect(node.Labels).To(SatisfyAll(
+			HaveKeyWithValue(v1.CPUTimerLabel+"tsc-frequency", "4008012000"),
+			HaveKeyWithValue(v1.CPUTimerLabel+"tsc-scalable", "false"),
```

Comment:
Ideally I'd like to avoid using unexported fields in test (and eventually change the package to nodelabeller_test), ergo the explicit values. I can create a getter for CPU counter if you prefer that style.

Reply:
> Ideally I'd like to avoid using unexported fields in test and eventually change the package to nodelabeller_test)

Even if this test was in another package, you've added `cpuCounter` as an argument for `NewNodeLabeller()`. In this test you're defining a `cpuCounter` struct with explicitly set values, then you're passing it to this constructor. So I don't really see how my suggestion uses unexported fields.

Reply:
Perhaps I misunderstood your suggestion, the scope of `cpuCounter` remains limited to the initialization function in which it is declared, therefore accessing it from `nlController` later on is equivalent to accessing an unexported field. 

I can either declare `cpuCounter` in the outer scope or introduce a getter method for the respective field in node-labeller, WDYT?

Reply:
Yeah I see your point.
If this value were to change sometime in the future, it'd be better to not change it in two places, so I think I would move `cpuCounter` out of the init function, make it global, and assign it in the BeforeEach block. But that's not a must from my end :)

Thanks for clarifying!

Reply:
Ack, done.